src/wallet/rpc-storage-interface.js

```diff
@@ -13,22 +13,25 @@ class RpcStorageInterface extends StorageInterface {
   constructor(directory) {
     super();
     this.directory = directory;
-    
+
     this.loadStorage(directory);
   }
-  
+
   async loadStorage(directory) {
     try {
       const data = await StorageRpc.getItem(directory);
       getLogger().log('retrieve data from rpc storage', data);
-      
+
       this.documents = JSON.parse(data);
 
       if (!this.documents) {
         this.documents = {};
       }
-    } catch(err) {
-      getLogger().log('error to retrieve data from rpc storage', err.toString());
+    } catch (err) {
+      getLogger().log(
+        'error to retrieve data from rpc storage',
+        err.toString(),
+      );
       this.documents = {};
```

Comment:
if we're swallowing the error but setting `.documents` to be empty, how does the calling code know the difference between, "there was nothing found" and, "something went wrong"?

Reply:
Should be fixed